bolt-sidecar/src/config.rs

```diff
@@ -63,9 +60,85 @@ pub struct Opts {
     /// new commitments for the next block (parsed as milliseconds)
     #[clap(short = 'd', long)]
     pub(super) commitment_deadline: Option<u64>,
-    /// The slot time in seconds
-    #[clap(short = 't', long)]
-    pub(super) slot_time_in_seconds: Option<u64>,
+    /// Chain on which the sidecar is running
+    #[clap(short = 'C', long, default_value = "mainnet")]
+    pub(super) chain: Chain,
+    /// Commitment signing options.
+    #[clap(flatten)]
+    pub(super) signing: SigningOpts,
+}
+
+/// Supported chains for the sidecar
+#[derive(Debug, Clone, ValueEnum)]
+#[clap(rename_all = "kebab_case")]
+#[allow(missing_docs)]
+pub enum Chain {
+    Mainnet,
+    Holesky,
+    Kurtosis,
+    Helder,
+}
+
+impl Chain {
+    /// Get the chain ID for the given chain.
+    pub fn chain_id(&self) -> u64 {
+        match self {
+            Chain::Mainnet => 1,
+            Chain::Holesky => 17000,
+            Chain::Kurtosis => 3151908,
+            Chain::Helder => 7014190335,
+        }
+    }
+
+    /// Get the chain name for the given chain.
+    pub fn name(&self) -> &'static str {
+        match self {
+            Chain::Mainnet => "mainnet",
+            Chain::Holesky => "holesky",
+            Chain::Kurtosis => "kurtosis",
+            Chain::Helder => "helder",
+        }
+    }
+
+    /// Get the slot time for the given chain in seconds.
+    pub fn slot_time(&self) -> u64 {
+        match self {
+            Chain::Mainnet => 12,
+            Chain::Holesky => 12,
+            Chain::Kurtosis => 2,
+            Chain::Helder => 12,
+        }
+    }
```

Comment:
I'd still keep the slot time as a separate CLI arg, with default 12, in case we want to run the devnet with a more realistic slot time. We can do it in a separate PR

Reply:
Yeah we should refactor the `Chain` configs to another file perhaps